act-rules-format.bs

```diff
@@ -15,9 +15,9 @@ Markup Shorthands: markdown yes
 Introduction {#intro}
 =====================
 
-There are currently many products available which aid their users in testing web content for conformance to accessibility standards such as the Web Content Accessibility Guidelines (WCAG) 2.0. As the web develops and grows in both size and complexity, these tools are essential for managing the accessibility of resources available on the web.
+There are currently many tools available which aid their users in testing web content for conformance to the Web Content Accessibility Guidelines (WCAG). As the web develops in both size and complexity, these tools are essential for managing the resources available on the web.
```

Comment:
@anthonyFer The following should not be removed "accessibility standards such as "